boards/BMS/Src/App/states/App_FaultState.c

```diff
@@ -13,6 +13,7 @@ static void FaultStateRunOnEntry(struct StateMachine *const state_machine)
     App_CanTx_BMS_Contactors_AirPositive_Set(
         App_Airs_IsAirPositiveClosed(airs) ? CONTACTOR_STATE_CLOSED : CONTACTOR_STATE_OPEN);
     App_OkStatus_Disable(bms_ok);
+    App_CanAlerts_SetFault(BMS_FAULT_STATE_FAULT, true);
```

Comment:
Aren't we already broadcasting the current state on line 11? Is this also necessary? Not sure how the new CAN alerts system works exactly

Reply:
CAN alerts works like if any boards sets a fault, then the BMS is supposed to open contactors. All the boards with fault states (DCM, FSM) set a "STATE_FAULT" alert when they enter fault state, which is what actually causes BMS to open contactors. So this is unnecessary on the BMS, since the BMS just checks for its own errors and doesn't check its CAN faults. I think I just did it so all the boards are the same. Was actually handy to only have to look at BMS_Faults msg while debugging though, so I think we should keep it.

Reply:
Oh okay makes sense